ahorcado.rb

```diff
@@ -1,62 +1,44 @@
-#TODO: word_spaces should depend from current_word
-# Array equivalent to the characters of the word 
-# [“”, “”, “”, “”, ...]
-# []
-# Array.new
-
-#current_word = "love"
-#word_spaces = Array.new(current_word.length, "")
-
-@current_word = "friends"
-@word_spaces = Array.new(@current_word.length, "")
-@current_characters = @current_word.split("")
-
-def display_game
-  @word_spaces.each do |characters|
-    print "| #{characters} | "
+class Ahorcado
+  def initialize(current_word)
+    @current_word = current_word
+    @word_spaces = Array.new(current_word.length, "")
+    @current_characters = current_word.split("")
+
+    display_game
+  end
+
+  def display_game
+    @word_spaces.each do |characters|
+      print "| #{characters} | "
+    end
+    choose_character
   end
-  choose_character
-end
 
-def word_spaces_empty?
-  @word_spaces.all? { |w| w == "" }
-end
+  def word_spaces_empty?
+    @word_spaces.any? { |w| w == "" }
+  end
```

Comment:
Can you think of other ways to handle evaluating empty spaces?

Reply:
Yes, I just push an update. Is that the change you were refering to?